cpp/oneapi/dal/detail/error_messages.cpp

```diff
@@ -83,6 +83,14 @@ MSG(object_does_not_provide_read_access_to_columns,
 MSG(object_does_not_provide_write_access_to_columns,
     "Given object does not provide write access to columns")
 MSG(unsupported_conversion_types, "Unsupported conversion types")
+MSG(row_indices_less_min_value, "Row indices are less than minimum acceptable value")
```

Comment:
```suggestion
MSG(row_indices_lt_min_value, "Row indices are less than minimum acceptable value")
```
please see the recommendations in the head of `error_messages.hpp` and apply them.

Reply:
Thanks, it was fixed